packages/angular/src/generators/init-mfe/lib/change-build-target.ts

```diff
@@ -0,0 +1,28 @@
+import type { Tree } from '@nrwl/devkit';
+import type { Schema } from '../schema';
+
+import {
+  readProjectConfiguration,
+  updateProjectConfiguration,
+} from '@nrwl/devkit';
+
+export function changeBuildTarget(host: Tree, options: Schema) {
+  const appConfig = readProjectConfiguration(host, options.appName);
+
+  appConfig.targets.build.executor = '@nrwl/angular:webpack-browser';
```

Comment:
What if the app was already using `@nrwl/angular:webpack-browser` with some custom webpack configuration?

Reply:
We'll replace the path to the configuration. Otherwise, everything else should remain the same.